driver/compile.go

```diff
@@ -33,6 +33,13 @@ type Config struct {
 	Warnings          chan string
 }
 
+func zbufDirInt(reversed bool) int {
+	if reversed {
+		return -1
+	}
+	return 1
+}
+
```

Comment:
Just a note: I think we have several different ways we specify sort direction. We should probably consolidate these at some point.

Reply:
Actually, should we change driver.MultiSource.OrderInfo() to return zbuf.Direction instead of bool?